src/rebar_prv_as.erl

```diff
@@ -38,7 +38,17 @@ do(State) ->
             {error, "At least one profile must be specified when using `as`"};
         _  ->
             State1 = rebar_state:apply_profiles(State, [list_to_atom(X) || X <- Profiles]),
-            rebar_prv_do:do_tasks(Tasks, State1)
+            {FirstTask, FirstTaskArgs} = hd(Tasks),
```

Comment:
I didn't manage to generate one except by using `as` as a namespace before forbidding it again in that branch. Worst case it silently does nothing:

```
λ /tmp → DEBUG=1 rebar3 as
===> Consult config file "rebar.config"
===> Consult config file "rebar.lock"
===> At least one profile must be specified when using `as`
λ /tmp → DEBUG=1 rebar3 as do
===> Consult config file "rebar.config"
===> Consult config file "rebar.lock"
λ /tmp → DEBUG=1 rebar3 as fake
===> Consult config file "rebar.config"
===> Consult config file "rebar.lock"
```